formatters/svg/svg.go

```diff
@@ -2,16 +2,57 @@
oZtyed8MYkx5NEq742KUeY/Ja4Q28266t/kB7LeCnawJ2vWsYA7iVnUsItZq39F+n7gEvcyZhgh2aOu6SFeqXWn9Fts1t6bLyb1GpA+F6xN76cHT9iU+idMm7f2EWljrNMveymavt75rBG7iS9YQBgdaMNJ8/z/cHxnUbLOwRa26u93JnvK+0RlTF7q/p8uqW3Op+CzYZ2Dn/6+MnIWtrgV9TwRsyZH76UeTCnVcqVUIwZiSjllRAz1TOQ/ldZ18u6BUGXORN8X/a2zNmeathvcGt4dyuqx5JPI6PfSNgXXjXl5SqV+ACiiG/B42mNgYNCBwiSGKYwVTFJMO5hjmCuYlzBfYGFgcWJJYmlgWcFyiFWLtYT1ClsU2yJ2OfYe9iMcARwnOH5xqnFaccZwFnDe4DrC9Ys7hnsR9yMeOZ4Qnn0833gNeMt4V/Fe4FPgS+LbwPeFP4p/nUCCwASBUwJfBA8JfhFSEQoSqhOaJnRI6IHQH2EF4XUiQiJpIqdEFUTnicmJZYhtEPsjbiCeJn5LQkuiTOKOpINkl+QnKS+pRVJnpCOku6QPSX+Q8ZJJkzkgawCEcbJX5MLkzskbyYfIv1OQUFinqKcYolijOEvJSSlOaYXSN+UY5SvKP1SiVGao7FN5p2qiWqO6RM1H7Zv6OQ0zjRKNLRo/NIs0L2h5aC3QltFO016j/UcnTGeezjNdM90+3U96JXpn9NX0+wykDNoMVhjKGWYYbjHiMEowmmX0xLjERMhkjWmAmZTZGfNZFiEWtyytLGdZsVjVWR2y5rAOsV5hI2ezzpbFNsH2iJ2f3Q67F/Zx9kccdBzmOIo4hjluceJxanIWco5z7nC+hAM+cH7n/MdFxEXHxcYlxqXGZZLLIZdvrnyuVkAY4JoBhMtcr7g+cBNwswIAdvGZQQAAAQAAAOgASgAFAAAAAAACAAEAAgAWAAABAAFXAAAAAHjanVXNThNRFD4tUH5EVoaFIeZqWGAipUVMlJ3/mhQxVnHjZtqZoYV2pnamNPgQxucwPoUrxBfQF/AJjGu/+93ToZTahZlM57v3nt/vnHMrIlfkj0xJbnpeRL7gdTgn17ByOC9Lcqp4Srbkh+JpKeYWFc/ISm5bcQF4T/Gs/M51Fc/JWj5UPC8r+Y+KF/K/8p8VX5K7M58UL8q7mRPFl+WgsKp4SbYKA90TWS58VfxNSoVBbKeyNDun+DvwssM/p+Tq7Ko8kqbs403xfpBAfDF4Paw9oLrE0pFj6VKqgV0ja9i9ie+mlKSM18hTSMU4b0HfyEPgLrTsr0e7sURSlAWeTLa2CfRKo3hG7VtAz6Ffh4UK9Go4HbZrZIffmHr70kMUHiTuSwKdACc+NYys451sYfKpGfGwR8kkk9tEhCU8ReQxztJkL01YMuQ85YmNuk0/h9iLJZzIjIFcwDomOAm48mnV2n4DiSqlXlLTspLSW0Sp12M87sJjCP06azqQrNO27Q1nOQZuKL8HYKbLCHzqDXJL4PliNcZ3SZXRHdHnDvftOuHZA6wTzctxZm30WIEI+5aPPqKxvhvEHjn1acF2XKTaNfSgmejLqK6ntYlYuSON9Ei9WZZD/ib0G8GHAfaYtWG0lpFwJApD1jzWwNW9jdOUsnXst/Ac6/S1wZHzWtP56nNaG1n+Vstx+gLf/vUbrPR5blwHhdqx1qu122VOZ4yus0o2n4BRWuTxNqhBo0W/Lq4G+8RjlQOtesrok6Fucb1hmIG1/pjR2jsgUGbf4u6ojLXo2BvuUluZFuNNhmxHjNbnXpwxbaVa6sll3OIddZhVKWTnOTZ9Wlv/B98huUnVa8yIfDyu7q7DYuj2WEU3Wa6v0wvMeeQ3Vr0Ob5JUY2lzUhrsw45sywaePp8iu3F4fuo6PUWNeeO/9WxcHTI4PCXdLJY2YqzoPRBl89cbmuRBJaq4jSq8OTraP0+UOTNiwc7O6O1Z5r15PgvXjU2sU8aTkMsic9jH+S48VNBDAVgOM19l3sM7areK0w6+t/lPVZI7sF3GahuonP3v3GM2Ifiyt3ua9U3AGTrLoyrvsdPEma1+6y+Qu2PYAAB42m3PR0xUcRDH8e/AsgtL79Xe+3tvWYp9F3j23rsosLuKgIurYkNjr9GY6Aliu6ix12jUgxp7iyXqwbM9HtSTBxfe35u/yyczyUxmiKA1fxqZzP/yESRCIonERhR2HEQTg5NY4ogngUSSSCaFVNJIJ4NMssgmh1zyaENb2tGeDnSkE53pQle60Z0e9KQXvelDX/qhoWPgIh83BRRSRDH9GcBABjGYIQzFg5cSSinDZBjDGcFIRjGaMYxlHOOZwEQmhe+fwlSmMZ0ZzGQWs5nDXOYxn3KxcZRNbOYGB8IfbWEPO2niOMckih28ZyP7xS4OdnOQbdzmg0TTzAl+8ZPfHOEUD7jHaRawkL1U8IhK7vOQZzzmCU/5RBUvec4LzuDjB/t4wyte4+cL39jOIgIsZgnV1HCIWpZSR5B6QixjOSv4zEpW0cBq1rKGqxymkXWsZwNf+c41znKO67zlncSIU2IlTuIlQRIlSZIlRVIlTdIlg/Nc4DJXuMNFLnGXrZyUTG5yS7Ikm12SI7mSZ/dVN9T5dUeoJqBpWqmlR1Oq2msoXcriFo3wgFJXGkqXMl/pVhYoC5VFyn/7PJa62qvrzqqALxSsrCiv91stw7R0m7ayULC2tXCbJS2aXuuOsMZftCyYdwAAAHjaRc6/DsFQFAbwe12qLaX/1CSpTXITi9Fg0S5dxNQmFvEOVhIxGFjtnuDUJF7AY/HhurbzO+c7J+fGn3viR5aROctLzk9FmRoy75NbZBTOUWyLHhlykTMScUJCTsmKk6s4V+QHJmCtFOqAOVYwgPpQoQYYF4UqUOsr2HFyZ4I7TLmBoX1QaAKN+AtOjnogeK84nGGlFOkaHR+5YKfpgf5S0wW9iWYbdAeaLbDtaYaf45vn/3gHgfChGYGdtWYXjEY/FhTKF1rTX4gAAA==`
```

Comment:
Can you put this in `font_liberation_mono.go` please, with appropriate license heading?

Reply:
defaults to font-family: `Consolas, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono, Bitstream Vera Sans Mono, Courier New, monospace` like before